client/js/README.md

```diff
@@ -1,52 +1,285 @@
-# `@gradio/client`
+## JavaScript Client Library
 
-A javascript client to call Gradio APIs.
+A javascript (and typescript) client to call Gradio APIs.
 
-**install it**
+## Installation
+
+The Gradio JavaScript client is available on npm as `@gradio/client`. You can install it as below:
```

Comment:
Once we've published, it would be really nice to have the CDN link here as well.

Reply:
Ah yeah, I'll figure that out as soon as possible.